frontend/public/config.yml

```
@@ -383,6 +383,35 @@
                   },
                 ],
             }
+      - label: 'Current LARP event'
+        name: 'current_larp_event'
+        file: 'frontend/src/content/pages/current-larp-event.md'
+        fields:
+          - { label: 'Najnovšia udalosť', name: 'current_larp_event', widget: 'relation', collection: 'events', searchFields: ['title'], valueField: 'title', displayFields: ['title'] }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)